family_foto/admin/admin.py

```diff
@@ -0,0 +1,26 @@
+from flask_admin import Admin
+
+from family_foto.admin.admin_approval_view import AdminApprovalView
+from family_foto.admin.admin_index_view import AdminHomeView
+from family_foto.admin.admin_model_view import AdminModelView
+from family_foto.admin.admin_promote_view import AdminPromoteView
+from family_foto.models import db
+from family_foto.models.file import File
+from family_foto.models.role import Role
+from family_foto.models.user import User
+
+
+def create_admin(app) -> None:
+    """
+    Creates the admin area
+    :param app: the current app needing an admin area
+    """
+    admin = Admin(app, name='FamilyFoto', template_mode='bootstrap4',
+                  index_view=AdminHomeView(url='/admin'))  # lgtm [py/call-to-non-callable]
+    admin.add_view(AdminModelView(User, db.session))  # lgtm [py/call-to-non-callable]
+    admin.add_view(AdminModelView(File, db.session))  # lgtm [py/call-to-non-callable]
+    admin.add_view(AdminModelView(Role, db.session))  # lgtm [py/call-to-non-callable]
+    admin.add_view(AdminApprovalView(name='Approval',  # lgtm [py/call-to-non-callable]
+                                     endpoint='approval'))  # lgtm [py/call-to-non-callable]
+    admin.add_view(AdminPromoteView(name='Promotion',  # lgtm [py/call-to-non-callable]
```

Comment:
The keyword argument is from one of its super classes.

Reply:
Thank you @Segelzwerg, for the feedback here.  We are looking hard at the pyre tool and thinking about how to take care of these cases where it is basically just misapplied or misconfigured.  If you have strong feelings on the matter then we're all ears.  Otherwise, happy to see you running Muse and we are working to limit this tool to a proper set of reports.

Reply:
@TomMD thank you for your reply. Configuration via a file would be nice. For example, LGTM is also complaining and it is a bit tedious to use comments for every line. I think pylint is taking a good approach where you can use a config file but also comment above a block.